csm-extensions/services/dev-postgres/chart/values.yaml

```diff
@@ -0,0 +1,26 @@
+---
+env:
+  CF_ADMIN_PASSWORD: ~
+  CF_ADMIN_USER: ~
+  CF_CA_CERT: ~
+  CF_DOMAIN: ~
+  SERVICE_LOCATION: ~
+  SERVICE_POSTGRESQL_HOST: ~
+  SERVICE_POSTGRESQL_PASS: cf-usb-sidecar-mysql-admin-password
```

Comment:
This refers to `mysql`? Is this intentional? If so, a comment here would clarify why.

Reply:
No, just find-and-replace failure. Fixed in 6d1a3aa5cdc71814d70ddd85aa7e9355ea748f06.